From 44d56ec2890ac3aba34e4f1a3aef54647356c63b Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:46 -0400
Subject: [redhat] kabi: Adding symbol unregister_shrinker (mm/vmscan.c)

Message-id: <b418fdc7dc291f31df33c7be8d4191d72c13b7b7.1410510773.git.poros@redhat.com>
Patchwork-id: 96086
O-Subject: [PATCH RHEL7.1 226/233] kabi: Adding symbol unregister_shrinker (mm/vmscan.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Used to unregister the shrinker call back function set in register_shrinker.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/unregister_shrinker
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/unregister_shrinker
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/unregister_shrinker

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/unregister_shrinker b/redhat/kabi/kabi-module/kabi_ppc64/unregister_shrinker
new file mode 100644
index 0000000..6da6101
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/unregister_shrinker
@@ -0,0 +1,2 @@
+#1-
+0xc50bab05 unregister_shrinker vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/unregister_shrinker b/redhat/kabi/kabi-module/kabi_s390x/unregister_shrinker
new file mode 100644
index 0000000..6da6101
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/unregister_shrinker
@@ -0,0 +1,2 @@
+#1-
+0xc50bab05 unregister_shrinker vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/unregister_shrinker b/redhat/kabi/kabi-module/kabi_x86_64/unregister_shrinker
new file mode 100644
index 0000000..6da6101
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/unregister_shrinker
@@ -0,0 +1,2 @@
+#1-
+0xc50bab05 unregister_shrinker vmlinux EXPORT_SYMBOL
-- 
1.7.1